docs/conf.py

```diff
@@ -28,7 +28,7 @@
 
     if not os.path.isdir(_themes):
         call(
-            [git, "clone", "git://github.com/Pylons/pylons_sphinx_theme.git", "_themes"]
+            [git, "clone", "https://github.com/Pylons/pylons_sphinx_theme.git", "_themes"]
```

Comment:
GitHub does no longer allow to use the unencrypted git protocol, see https://github.blog/2021-09-01-improving-git-protocol-security-github/

Reply:
We have made our pylons sphinx theme a pypi project, @stevepiercy can probably figure out what changes need to happen there, instead of calling git clone.

Reply:
I would suggest to update the document creation in a separate PR, as Read the Docs is sometimes a bit ... well... touchy :-)